src/modules/menu/components/activeLayers/MenuActiveLayersListItemTimeSelector.vue

```diff
@@ -60,13 +76,26 @@ export default {
             return timestamps
         },
     },
+    mounted() {
+        if (this.hasMultipleTimestamps) {
+            this.popover = tippy(this.$refs.timeSelectorButton, {
+                theme: 'popover-button light-border',
+                content: this.$refs.timeSelectorModal,
+                allowHTML: true,
+                placement: 'right',
+                interactive: true,
+                arrow: true,
+                trigger: 'click',
+            })
```

Comment:
Did I missed where you delete the tippy instance ?

Reply:
🙈 nope